### DISCUSSION TITLE
[Bug] 创建的v3 egg项目在tsconfig.json中设置 "baseUrl": "./",  //     "paths": {       "@/*": ["src/*"],      },会导致无法打包

### DISCUSSION THREAD

**Discussion by sunny-fun** [midwayjs/midway#3496] [Q&A]
### Describe the problem（描述问题）

```json
{
  "compileOnSave": true,
  "compilerOptions": {
    "baseUrl": "./",  
    "paths": {
      "@/*": ["src/*"], 
    },
    "target": "es2018",
    "module": "commonjs",
    "moduleResolution": "node",
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true,
    "inlineSourceMap":true,
    "noImplicitThis": true,
    "noUnusedLocals": true,
    "stripInternal": true,
    "skipLibCheck": true,
    "pretty": true,
    "declaration": true,
    "forceConsistentCasingInFileNames": true,
    "typeRoots": [ "./typings", "./node_modules/@types"],
    "outDir": "dist"
  },
  "exclude": [
    "dist",
    "node_modules",
    "test"
  ]
}

```

tsconfig配置如上 添加
```
 "baseUrl": "./",  
    "paths": {
      "@/*": ["src/*"], 
    },
```
后打包会报
> yarn run build     
Debugger attached.
yarn run v1.22.21
$ midway-bin build -c
Debugger attached.
[ Midway ] Read TsConfig Error

Error! You can try adding the -V parameter for more information output.

Error: [ Midway ] Read TsConfig Error

### Midway Versions（Midway 版本）

  "dependencies": {
    "@midwayjs/bootstrap": "^3.12.0",
    "@midwayjs/core": "^3.12.0",
    "@midwayjs/web": "^3.12.0",
    "@midwayjs/decorator": "^3.12.0",
    "@midwayjs/logger": "^2.14.0",
    "egg": "^2.0.0"
  },
  "devDependencies": {
    "@midwayjs/cli": "^2.0.0",
    "@midwayjs/egg-ts-helper": "^1.0.1",
    "@midwayjs/mock": "^3.12.0",
    "@types/jest": "^29.2.0",
    "@types/node": "14",
    "cross-env": "^6.0.0",
    "egg-mock": "^3.26.0",
    "mwts": "^1.3.0",
    "jest": "^29.2.2",
    "ts-jest": "^29.0.3",
    "typescript": "~4.8.0"
  },

**Comment by czy88840616** [comment]
只是你 tsconfig 不是标准 json，肯定是哪里多了逗号

**Reply by sunny-fun** [reply]
感谢感谢
